src/hyperlane-adapter/HyperlaneReceiverAdapterV2.sol

```diff
@@ -6,16 +6,17 @@ import { IMailbox } from "./interfaces/IMailbox.sol";
 import { IMessageRecipient } from "./interfaces/IMessageRecipient.sol";
 import { IInterchainSecurityModule, ISpecifiesInterchainSecurityModule } from "./interfaces/IInterchainSecurityModule.sol";
 import { TypeCasts } from "./libraries/TypeCasts.sol";
+import { EncodeDecodeUtil } from "./libraries/EncodeDecodeUtil.sol";
 import { Errors } from "./libraries/Errors.sol";
-import { IMessageDispatcher } from "./interfaces/IMessageDispatcher.sol";
-import { IMessageExecutor } from "./interfaces/IMessageExecutor.sol";
-import "./libraries/MessageLib.sol";
+import { IMessageDispatcher } from "../interfaces/IMessageDispatcher.sol";
+import { IMessageExecutor } from "../interfaces/IMessageExecutor.sol";
+import "../libraries/MessageLib.sol";
 
 /**
  * @title HyperlaneReceiverAdapter implementation.
  * @notice `IBridgeReceiverAdapter` implementation that uses Hyperlane as the bridge.
  */
-contract HyperlaneReceiverAdapter is
+contract HyperlaneReceiverAdapterV2 is
```

Comment:
updated

Reply:
Should this be v3?

Reply:
I have updated the adapter to use [hyperlane V3](https://github.com/degencodebeast/ERC5164/commit/d778c87c8ddea150cb562f7d88631786b9cf555f#r137662746)